graders/r/Dockerfile

```diff
@@ -84,3 +84,7 @@ RUN useradd ag \
 
 # Copy autograder interface script into position
 COPY r_autograder /r_autograder
+
+# Install plr (helper package for grading R code in PrairieLearn)
+COPY plr /r_autograder/plr
+RUN R CMD INSTALL r_autograder/plr
```

Comment:
We could make this line 75 instead and write `RUN install.r r_autograder/plr` to use the corresponding script from littler but that is really a marginal change.

Reply:
Done.